numpy/core/fromnumeric.py

```diff
@@ -2647,9 +2647,9 @@ def ptp(a, axis=None, out=None, keepdims=np._NoValue):
 
     Returns
     -------
-    ptp : ndarray
-        A new array holding the result, unless `out` was
-        specified, in which case a reference to `out` is returned.
+     ptp : ndarray or scalar
```

Comment:
Just a nit - let's get rid of the extra space that was added.
```suggestion
    ptp : ndarray or scalar
```